hw/ip/edn/data/edn.hjson

```diff
@@ -351,6 +339,26 @@
                 '''
           resval: "0"
         }
+        { bits: "4:3",
+          name: "CMD_STS",
+          desc: '''
+                This field represents the status code returned with the CSRNG application command ack.
+                It is updated each time a SW command is acknowledged by CSRNG.
+                To check whether a command was successful, wait for !!INTR_STATE.EDN_CMD_REQ_DONE or
+                !!SW_CMD_STS.CMD_ACK to be high and then check the value of this field.
+                0b00: Request completed successfully.
+                0b01: Request completed with an invalid application command error.
+                      This error indicates that the issued application command doesn't represent a valid operation.
+                      If this error appears, the main state machine will hang and the entropy complex has to be restarted.
+                0b10: Request completed with an invalid state parameter error.
+                      This error indicates that the state wasn't zeroized properly after an uninstantiate command.
+                      In this case the entropy complex should be restarted to properly zeroize the state.
+                0b11: Request completed with an invalid counter drbg generation command error.
+                      This error indicates that CSRNG entropy was generated for a command that is not a generate command.
+                      In this case the entropy should not be considered as valid.
```

Comment:
Instead of duplicating this information from CSRNG, would it makes sense to refer to the CSRNG doc (like for `HW_CMD_STS.CMD_TYPE` below)?

Reply:
Yes it would.